README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+
+---
+#"Computing for Social Sciences HW01"
+"**Taylor Griffin**"  
+"*October 8, 2019*"  
+
+# Introduction
+
+![Taylor Griffin](C:/Users/Taylor Griffin/Desktop/Computing R/Taylor G hw01/Taylor.png)
+
+My name is Taylor Griffin and I am a second year student at the Harris School of Public Policy. Prior to Harris,I served as a Teach for America Corp member.As a Teach for America Corps member, I was challenged to think critically about **policies and practices** that **disproportionately harmed** already vulnerable communities. I realized that access and opportunity do not look the same in every community. If we compare educational inequity to a foot race, all students would participate in the competition, but would not line up at the same starting point. Thus, the fight towards achieving educational equity drives my goal: *to increase access to excellent and equitable education for students of color in low-income communities*. While attending the Harris School of Public Policy, I've become a critical consumer and careful producer of descriptive statistics. The Harris School curriculum has strengthened my quantitative skills and pushed me to provide *data-driven answers to complex questions*.
+
+## Goals for the Quarter
+1. Learn and understand the syntax of different packages in R
+2. Learn how to troubleshoot errors and research solutions
+3. Learn how to clean and manipulate data
+4. Become proficient in data visualization packages like ggplot2  
+
+## Programming Courses
+Previous Courses | Future Courses
+------------ | -------------
+Econometrics I & II | Data Skills for Public Policy
+Introduction to Programming in Python | Machine Learning  
```

Comment:
It shows your mastery of Markdown to insert this good-looking table! I am definitely learning from you :)